js/layout-javascript/agenda-code.js

```diff
@@ -187,8 +187,8 @@ DynamicList.prototype.attachObservers = function() {
   // Attach your event listeners here
   $(window).resize(function() {
     _this.centerDate();
-
     _this.Utils.DOM.adjustAddButtonPosition(_this);
+    _this.correctPosition();
```

Comment:
Can we be more specific about what position this is correcting? Otherwise it's as generic as "runCalculations" or "handleData". You can see that "centerDate" and "adjustAddButtonPosition" are both pretty specific about what they do.